util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1136,9 +1146,10 @@ static bool HashSha3(const Span<const uint8_t> args[], ReplyCallback write_reply
 
 template <const EVP_MD *(MDFunc)()>
 static bool HashXof(const Span<const uint8_t> args[], ReplyCallback write_reply) {
-  // NOTE: Max outLen in the test vectors is 1024 bits (128 bytes). If that
+  // NOTE: Max outLen supported by ACVP is 65536 bits (8192 bytes). If that
```

Comment:
Can we test/assert that somewhere?

Reply:
It's asserted in the ACVP specifications for capabilities registration: https://pages.nist.gov/ACVP/draft-celi-acvp-sha3.html#name-sha3-and-shake-algorithm-ca